<?xml version="1.0" encoding="UTF-8"?>
<testsuites tests="2" failures="2" disabled="0" errors="0" timestamp="2018-04-10T21:35:20" time="0.205" name="AllTests">
  <testsuite name="GarbageCollectionTests" tests="2" failures="2" disabled="0" errors="0" time="0.205">
    <testcase name="UpdateTest" status="run" time="0.204" classname="GarbageCollectionTests">
      <failure message="unknown file&#x0A;C++ exception with description &quot;Database UPDATE_DB is not found&quot; thrown in the test body." type=""><![CDATA[unknown file
C++ exception with description "Database UPDATE_DB is not found" thrown in the test body.]]></failure>
    </testcase>
    <testcase name="DeleteTest" status="run" time="0.001" classname="GarbageCollectionTests">
      <failure message="unknown file&#x0A;C++ exception with description &quot;Database DELETE_DB is not found&quot; thrown in the test body." type=""><![CDATA[unknown file
C++ exception with description "Database DELETE_DB is not found" thrown in the test body.]]></failure>
    </testcase>
  </testsuite>
</testsuites>
